common/src/Microsoft.Azure.IIoT.Core/src/Diagnostics/Default/MetricServerHost.cs

```diff
@@ -0,0 +1,46 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.Messaging.Default {
```

Comment:
Namespace name.  And empty line after.   

However, as below, please move this to AspNetCore project which is already being referenced by the processors.